pkg/volume/volume.go

```diff
@@ -103,6 +103,9 @@ type Attributes struct {
 
 // MounterArgs provides more easily extensible arguments to Mounter
 type MounterArgs struct {
+	// When FsUser is set, the ownership of the volume will be modified to be
+	// owned and writable by FsUser. Otherwise, there is no side effects.
```

Comment:
Add a comment that this is currently only supported with projected service account tokens

Reply:
done